code/game/objects/items/food/egg.dm

```diff
@@ -101,7 +101,7 @@
 
 	var/obj/machinery/griddle/hit_griddle = target
 	hit_griddle.AddToGrill(broken_egg, user)
-	target.balloon_alert(user, "You crack [src] open onto [target].")
+	target.balloon_alert(user, "cracks [src] open")
```

Comment:
this isn't the right tense, is it?

Reply:
What's wrong with present tense? Past tense seems weird for an instant ephemeral alert